### DISCUSSION TITLE
Exporting multiple comps from one AE file

### DISCUSSION THREAD

**Discussion by CraftedbyRonin** [inlife/nexrender#972] [Q&A]
Can you export multiple comps from the same template file without the program needing to load up the ae file and dependencies separately for each export?

**Comment by inlife** [comment]
Hey @CraftedbyRonin 
The default behavior does not allow that. However, that could potentially be achieved with some custom scripting. But that is just a theory.

**Reply by CraftedbyRonin** [reply]
Alright, thanks!
